### ISSUE TITLE
Configurable documentroot?

### ISSUE THREAD

**Issue by andrewklau** [sclorg/s2i-php-container#107]
It would be desirable if we could have the DocumentRoot configurable as an environment variable, ie. our repository is configured to have composer.json in the root directory, but a different directory stores the public content (ie. /web or /public)

**Comment by andrewklau** [comment]
Perhaps we could try define something like this?

```
<IfDefine pub>
  Define documentroot /pub
</IfDefine>
<IfDefine web>
  Define documentroot /web
</IfDefine>
DocumentRoot "/opt/app-root/src${documentroot}"
```

changing the run to:
`httpd -D frontend -D ${DOCUMENTROOT}`

**Comment by hhorak** [comment]
Fixed by #111.
